src/Classes/ConfigTab.lua

```diff
@@ -265,6 +279,26 @@ local ConfigTabClass = newClass("ConfigTab", "UndoHandler", "ControlHost", "Cont
 					end
 					return innerLabel
 				end
+				local innerTooltipFunc = control.tooltipFunc
```

Comment:
looks like merge/rebase went wrong, tooltipFunc logic is here twice

Reply:
Thanks for checking. Should be fixed by [2070ff3](https://github.com/PathOfBuildingCommunity/PathOfBuilding/pull/5074/commits/2070ff37d0a5035926a9542f45dcd48110fc60cc)